addons/sale_project/models/project.py

```diff
@@ -17,6 +17,17 @@ class Project(models.Model):
     _inherit = 'project.project'
 
     allow_billable = fields.Boolean("Billable")
+    billing_type = fields.Selection(
+        compute="_compute_billing_type",
+        selection=[
+            ('manually', 'billed manually'),
+            ('not_billable', 'not billable'),
+        ],
+        default='not_billable',
+        required=True,
+        readonly=False,
+        store=True,
```

Comment:
Requires a migration script.

Reply:
to set not_billable for existing projects ? if yes, isn't the default and required = True enough ?

Reply:
To directly create the column with the default value instead of letting the ORM doing that. 🙂